onebusaway-android/src/google/java/org/onebusaway/android/map/googlemapsv2/BaseMapFragment.java

```diff
@@ -738,6 +738,7 @@ public void onLocationChanged(Location l) {
         if (firstFix) {
             // If its our first location (across all BaseMapFragments), move the camera to it
             setMyLocation(true, false);
+            onFocusChanged(null,null,l);
```

Comment:
@barbeau Yes, my bad. Probably when I was merging the previous pr I missed this. I think the new PR will help to prevent similar problems in the future.